js/modules/checkUsernamePassword.js

```diff
@@ -0,0 +1,15 @@
+import { getUsers } from "./saveToLocalStorage.js";
+import { encryptPassword } from './encryptPassword.js'
+
+export function checkUserAndPassword(username, password) {
+    const savedUsers = getUsers(); 
+    let checkUser = savedUsers.find((user) => {
+        return username == user.username
+    });
+    
+    if (checkUser || checkUser.password != encryptPassword(password)) {
```

Comment:
you have to add a negation to checkUser for this one to work like that, you are trying to check for not existent checkUsers

```suggestion
    if (!checkUser || checkUser.password != encryptPassword(password)) {
```